simpleflow/swf/executor.py

```diff
@@ -798,8 +798,16 @@ def resume(self, a_task: ActivityTask | WorkflowTask | SignalTask | MarkerTask,
             self.schedule_task(a_task, task_list=self.task_list)
             future = futures.Future()  # return a pending future.
 
-        if self._open_activity_count == constants.MAX_OPEN_ACTIVITY_COUNT:
-            logger.warning(f"limit of {constants.MAX_OPEN_ACTIVITY_COUNT} open activities reached")
+        max_open_activity_count = (
+            getattr(self.workflow, "max_open_activity_count", None) or constants.MAX_OPEN_ACTIVITY_COUNT
```

Comment:
Does this mean if we set `max_open_activity_count` to 0, it's ignored and we use `constants.MAX_OPEN_ACTIVITY_COUNT`? As long as it's the intended behaviour, I'm ok. Otherwise an explicit check for None, or a `try/except AttrributeError` might be more precise.